### PR TITLE
Nightly ribbon JS

### PR DESCRIPTION
**Pull request by lzap**
This adds nightly ribbon to the top-right corner for HTML builds (only). It is implemented via docinfo feature of asciidoctor which allows me not to edit the templates. It's purely based on javascript and injected into HTML head, if "nightly" or "file://" is found in an URL, ribbon is added to the page.

### CODE REVIEW

**Comment by lzap** [comment]
This is how it looks like:

image

**Comment by lzap** [comment]
Rebasing, there was a leftover from my previous checkout.

File: guides/doc-Administering_Red_Hat_Satellite/docinfo.html [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/lzap/work/foreman-documentation/guides/common/docinfo.html
```
Comment (ekohl) [inline guides/doc-Administering_Red_Hat_Satellite/docinfo.html:1]: This looks incorrect

Comment (lzap) [reply guides/doc-Administering_Red_Hat_Satellite/docinfo.html:1]: Haha yeah Midnight Commander.

Comment (lzap) [reply guides/doc-Administering_Red_Hat_Satellite/docinfo.html]: Rebased to relative links.

**Review by melcorr — APPROVED** [review]
Looks lovely.

**Comment by lzap** [comment]
> Looks lovely.

I've used some bad words when creating it, yeah, lovely. CSS is not my thing. :-)

**Pull request merged**